sycl/include/sycl/detail/pi.h

```diff
@@ -434,7 +436,9 @@ typedef enum {
   PI_COMMAND_TYPE_SVM_MEMCPY = 0x120A,
   PI_COMMAND_TYPE_SVM_MEMFILL = 0x120B,
   PI_COMMAND_TYPE_SVM_MAP = 0x120C,
-  PI_COMMAND_TYPE_SVM_UNMAP = 0x120D
+  PI_COMMAND_TYPE_SVM_UNMAP = 0x120D,
+  PI_COMMAND_TYPE_DEVICE_VARIABLE_READ = 0x418E,
```

Comment:
```suggestion
  PI_COMMAND_TYPE_DEVICE_GLOBAL_VARIABLE_READ = 0x418E,
```

?

Reply:
The naming is based on the interfaces specified in the [design document](https://github.com/intel/llvm/blob/sycl/sycl/doc/design/DeviceGlobal.md#new-pi-interface-to-copy-to-or-from-a-module-scope-variable). That said, I agree that the current name is a little confusing, so if people agree I will go change the naming.

Reply:
I suppose we should tag @gmlueck here

Reply:
Yes, I agree that the proposed names are better.